doc/user/data/explain_plan_operators.yml

```diff
@@ -208,6 +356,26 @@ operators:
     expansive: False
     example: "`Union`"
 
+  - operator: Union
+    plan_types: "lir"
+    description: |
+      Combines its inputs into a unified output, emitting one row for each row on any input.
+    uses_memory: True
+    memory_details: |
+      If the union "consolidates output", it will make moderate use of memory, particularly at hydration time. If the union is not marked with "consolidates output", it will not consume memory.
+    expansive: False
+    example: "`Union 7 10 11 14 (consolidates output)`"
+
+  - operator: ArrangeBy
+    plan_types: "optimized"
```

Comment:
Should this be `"lir"`?

Reply:
No---we render them slightly differently in LIR. I had this entry duplicated for some reason, though. 🤔